playground.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  extends: [
+    "airbnb",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "./rules/eslint.js",
+    "./rules/react-hooks.js",
+    "./rules/react.js",
+    "./rules/promise.js",
+    "./rules/import.js",
+    "./rules/prettier.js"
+  ],
+  plugins: [
+    "@typescript-eslint"
+  ],
+  env: {
```

Comment:
These can be project specific, can be removed and added to the project's `.eslintrc`

Reply:
moved 👍